### DISCUSSION TITLE
Introduce yourself and your projects

### DISCUSSION THREAD

**Discussion by formidablae** [formidablae/formidablae#32] [Show yourself and your projects]
## 👋 Hello and welcome!
To get started, comment below with an introduction of yourself, something like who you are, where you are from and what do you do in life. If you fancy, you can share what interesting projects you have worked or are currently working on :smile:.

**Comment by ashishpaymode** [comment]
Hii Brother,
 Can you please help me?
 Can you please make this Angular project?

**Reply by formidablae** [reply]
Hello @ashishpaymode, pleased to have you here. How can I help you (with an Angular project)?
